io.catenax.request_for_quotation/2.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,577 @@
+
+:FilePathCharacteristic a samm:Characteristic ;
+   samm:preferredName "file path characteristic"@en ;
+   samm:description "path or address to the file "@en ;
+   samm:dataType xsd:string .
+
+:FileNameCharacteristic a samm:Characteristic ;
+   samm:preferredName "file name characteristic"@en ;
+   samm:description "name of the file"@en ;
+   samm:dataType xsd:string .
+
+:FileObjectCharacteristic a samm:Characteristic ;
+   samm:preferredName "file object characteristic"@en ;
+   samm:description "the file object itself, in case this could be sent via the API"@en ;
+   samm:dataType :File .
+
+:PartIdCharacteristic a samm:Characteristic ;
+   samm:preferredName "part Id Characteristic"@en ;
+   samm:description "identifier of the part from CAD File"@en ;
+   samm:dataType xsd:string .
+
+:PartNameCharacteristic a samm:Characteristic ;
+   samm:preferredName "part name characteristic"@en ;
+   samm:description "name of the part in CAD file. In case the CAD file has only one part, this could also be the file name"@en ;
+   samm:dataType xsd:string .
+
+:ManufacturingDomainCharacteristic a samm:Characteristic ;
+   samm:preferredName "manufacturing domain characteristic"@en ;
+   samm:description "domain of required process (e.g. subtractive manufacturing, additive manufacturing, assembly)"@en ;
+   samm:dataType xsd:string .
+
+:BillOfProcessCharacteristic a samm:Characteristic ;
+   samm:preferredName "bill of process characteristic"@en ;
+   samm:description "list of processes to manufacture the part(s)"@en ;
+   samm:dataType :BillOfProcessReference .
+
+:SetOfProcessStepsCharacteristic a samm:Characteristic ;
+   samm:preferredName "set of process steps characteristic"@en ;
+   samm:description "depending on the manufacturing domain there are several process steps to be done. These process steps might also be hierarchical."@en ;
+   samm:dataType :ProcessSteps .
+
+:GeneralToleranceCharacteristic a samm:Characteristic ;
+   samm:preferredName "general tolerance characteristic"@en ;
+   samm:description "selection of defined tolerance classes"@en ;
+   samm:dataType xsd:string .
+
+:AdditionalQualityRequirementsCharacteristic a samm:Characteristic ;
+   samm:preferredName "additional quality requirements characteristic"@en ;
+   samm:description "specific requirements to guarantee quality"@en ;
+   samm:dataType xsd:string .
+
+:MaterialCharacteristic a samm:Characteristic ;
+   samm:preferredName "material characteristic"@en ;
+   samm:description "material definition with further information"@en ;
+   samm:dataType :MaterialData .
+
+:PartQuantityCharacteristic a samm:Characteristic ;
+   samm:preferredName "part quantity characteristic"@en ;
+   samm:description "Describes the quantity in which the part item is assembled in the given parent item by providing a quantity value and the measurement unit in which the quantity is measured."@en ;
+   samm:dataType :Quantity .
+
+:NumberOfObjects a samm:Characteristic ;
```

Comment:
please also add `preferredName` here

Reply:
Preferred name for `NumberOfObjects` has been added